README.md

```diff
@@ -28,17 +28,38 @@ See the Kanban [board](https://github.com/bob-cd/bob/projects/1) to see the road
 ## Configuration
 Configuration uses the [environ library](https://github.com/weavejester/environ) and therefore several variables can be
 set by specifying them as environment variable or as java system property. Possible variables are:
-| java system properties | environment variables | defaults                    |
-|------------------------|-----------------------|-----------------------------|
-| bob-db-host            | BOB_DB_HOST           | localhost                   |
-| bob-db-port            | BOB_DB_PORT           | 5432                        |
-| bob-db-user            | BOB_DB_USER           | bob                         |
-| bob-db-name            | BOB_DB_NAME           | bob                         |
-| bob-docker-uri         | BOB_DOCKER_URI        | unix:///var/run/docker.sock |
-| bob-connect-timeout    | BOB_CONNECT_TIMEOUT   | 1000ms                      |
-| bob-read-timeout       | BOB_READ_TIMEOUT      | 30000ms                     |
-| bob-write-timeout      | BOB_WRITE_TIMEOUT     | 30000ms                     |
-| bob-call-timeout       | BOB_CALL_TIMEOUT      | 40000ms                     |
+| java system properties     | environment variables      | defaults                    |
+|----------------------------|----------------------------|-----------------------------|
+| bob-postgres-host          | BOB_POSTGRES_HOST          | localhost                   |
+| bob-postgres-port          | BOB_POSTGRES_PORT          | 5432                        |
+| bob-postgres-user          | BOB_POSTGRES_USER          | bob                         |
+| bob-postgres-name          | BOB_POSTGRES_NAME          | bob                         |
```

Comment:
can we call this `BOB_POSTGRES_DB_NAME`?

Reply:
database would be consistent with my other namings. changed it...